src/main/java/javaserver/ServerFactory.java

```diff
@@ -9,9 +9,9 @@
 import java.net.Socket;
 
 public class ServerFactory {
+
 	public Server createServer(int port) throws IOException {
```

Comment:
You can declare these methods `static` for your factory class since you are simply returning new instances of objects, I also feel like this class's name is misleading as it isn't just creating servers.